Modules/IO/NIFTI/test/itkNiftiReadWriteDirectionTest.cxx

```diff
@@ -49,28 +49,34 @@ itkNiftiReadWriteDirectionTest(int ac, char * av[])
   };
 
   using TestImageType = itk::Image<float, 3>;
-  TestImageType::Pointer LPSLabel = reader_lambda(av[1]);
-  TestImageType::Pointer LPSLabel_noqform = reader_lambda(av[2]);
-  TestImageType::Pointer LPSLabel_nosform = reader_lambda(av[3]);
+  TestImageType::Pointer InputImage = reader_lambda(av[1]);
```

Comment:
It would be good if this commit was squashed with the previous one.

Reply:
I've squashed the commits into one

Reply:
:+1: Done in ceac451.